andela/might-guy-cfh
# #141409257 Send Invites to at Most Eleven People

#### What does this PR do?
Implements the feature 'Users should be able to send invites to at most 11 people other than themselves'.

#### Description of Task to be completed?
- Users should be able to invite people to their game up to a maximum of 11 other people, so no game can exceed 12 people in total and must have at least 3 players.
- GIVEN has created a game and is inviting friends
- WHEN user wants to start the game and there are less than 3 people
- THEN a popup shows up explaining a minimum of 3 players are required to play the game
- WHEN 11 people have been invited, when the 12th user is to be added
- THEN a popup shows up explaining only a maximum of 12 players are allowed per game
- An authenticated endpoint to search current users would be required at /api/search/users

#### How should this be manually tested?
- Visit the review app on [Might-Guy-Cfh](http://might-guy-cfh-staging-pr-31.herokuapp.com/)
- Signup/log in 
- Choose to play with friends
- If you try to start a game with less than three(3) players, a pop-up appears informing you that you need at least three(3) players to start the game
- Click on the **Invite Players** button to invite friends
- You can only invite Eleven(11) players. If you try to exceed that, you get a message informing you that you have reached the limit of invites.

#### Any background context you want to provide?
A user had to copy the invite link and send to friends on different media. Now, a user can send invite via email to players, without having to leave the app

#### What are the relevant pivotal tracker stories?
Users should be able to send invites to at most 11 people other than themselves

#### Screenshots (if appropriate)
NA
#### Questions:
NA
